.gitignore

```diff
@@ -7,3 +7,7 @@
 /spec/reports/
 /tmp/
 /vendor/
+
+# RubyMine files
+.idea/
+.rakeTasks
```

Comment:
It's often a good idea to put these in your personal global git ignore file located at `~/.gitignore`.